client/src/js/components/bhMultipleDebtorGroupSelect.js

```diff
@@ -0,0 +1,51 @@
+angular.module('bhima.components')
```

Comment:
@jniles 
I have noticed this : 
* The two components interfaces are : 
for ```bhDebtorGroupSelect```
````
bindings    : {
      debtorGroupUuid   : '<',
      disable           : '<?',
      onSelectCallback  : '&',
      name              : '@?',
      label             : '@?',
      required          : '<?',
      validationTrigger : '<',
    }
```

For bhMultipleDebtorGroup

```
bindings    : { 
      label            : '@?',
      onSelectCallback : '&',
      onRemoveCallback : '&',
      formName         : '@?'
    }
```
* The controller can be combined without problem
* The views (html templates) are pretty different, the ```bhMultipleDebtorGroupSelect``` does not close the select after a selection, and the ```on-select``` event is firing a function which receive a model (not ```$item``` like bhDebtorGroupSelect).

The easiest solution is to create an issue so I can do it separately or I will need some guidance.
What do you think?